app/views/items/show.html.erb

```diff
@@ -1,17 +1,27 @@
 <% content_for :sidebar do %>
-  <%= sidebar_button_link 'Return to Listing', items_path %>
   <%= edit_item_button %>
   <%= add_repair_button %>
   <%= add_inspection_button %>
 <% end %>
 
 <h3>
+  <%= link_to @item.item_category,
+                item_category_path(@item.item_category) %> >
+  <%= link_to @item.item_type, @item.item_type %> >
```

Comment:
I'm dropping the `.name` everywhere, so if I want to change the label, I can do it in one place. 
The form I have is a valid way to specify the target, but I do have it the other way in several places. Also, I am trying to encourage more junior developers, so perhaps the longer version would be better.

Changed